src/api/spec/factories/notification.rb

```diff
@@ -6,5 +6,14 @@
   end
 
   factory :rss_notification, parent: :notification, class: 'Notification::RssFeedItem' do
+    transient do
+      stale { false }
+    end
+    after(:create) do |notification, evaluator|
+      if evaluator.stale
+        notification.created_at = 6.months.ago if evaluator.stale
```

Comment:
`evaluator.stale` is being checked twice. It seems like a left over, isn't it?

Reply:
done